packages/lesswrong/lib/sql/AddFieldQuery.ts

```diff
@@ -8,16 +9,15 @@ import Table from "./Table";
  * meant for use in migrations.
  */
 class AddFieldQuery<T extends DbObject> extends Query<T> {
-  constructor(table: Table<T>, fieldName: string, skipValidation?: boolean) {
```

Comment:
Just checking, you removed `skipValidation` because it wasn't being used?

Reply:
Yep